test/es6/rlexe.xml

```diff
@@ -1331,4 +1331,12 @@
     <compile-flags>-force:deferparse -args summary -endargs</compile-flags>
   </default>
 </test>
+<test>
+  <default>
+    <files>bug_issue_1496.js</files>
+    <compile-flags>-mic:1 -maxsimplejitruncount:2</compile-flags>
```

Comment:
Let's have dynapogo run the test. More aggressive jitting in such a case can expose issues with inlining, etc., down the road.